client/app/telecom/pack/xdsl/modem/templates/pack-xdsl-modem-templates.html

```diff
@@ -0,0 +1,82 @@
+<section class="widget-presentation table-router">
+    <header class="widget-presentation-header">
+        <h2 class="widget-presentation-title" data-translate="xdsl_modem_template_title"></h2>
+    </header>
+
+    <div data-ng-if="$ctrl.message">
```

Comment:
```suggestion
    <oui-message
      data-type="{{ $ctrl.message.type }}"
      data-ng-if="$ctrl.message">
        <span data-ng-bind="$ctrl.message.detail"></span>
    </oui-message>
```

Reply:
Sorry, but your suggestion doesn't work.

Reply:
Well what doesn't work ? :) We can certainly make it work

Reply:
data-type doesn't understand $ctrl.message.type to apply associated style for oui-message